crates/ruma/Cargo.toml

```diff
@@ -284,6 +281,13 @@ __ci = [
     "unstable-msc4140"
 ]
 
+# Private feature, only used in test / benchmarking code
```

Comment:
This comment should go above `__unstable-mscs` and change the beginning to `Private features`, plural. Also no need to add an empty line between the declarations of `__unstable-mscs` and `__ci`.